scripts/tools/eslint-plugin-prettier-internal-rules/prefer-ast-path-getters.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const selector = [
+  "CallExpression[arguments.length<2]",
+  "> MemberExpression",
+].join("");
+
+const messageId = "prefer-ast-path-getters";
+
+const getNodeFunctionNames = new Set(["getValue", "getNode"]);
+const getParentNodeFunctionName = "getParentNode";
+
+function isPathMemberExpression(node) {
+  return node.object?.name?.toLowerCase().endsWith("path");
```

Comment:
MemberExpression's object can't be nullish.

Reply:
If we add `Identifier` check to the `MemeberExpression` selector, then the name can't be nullish either.